R/model-modules.R

```diff
@@ -247,6 +241,19 @@ enw_report <- function(non_parametric = ~0, structural = ~0, data) {
     prefix = "rep", drop_intercept = TRUE
   )
 
+  # Check for structural model and define
+  if (as_string_formula(structural) != "~0") {
+    cli::cli_alert_warning(
+      "The structural reporting model is in experimental development"
+    )
+    data_list$agg_probs <- 1
+    # "structural" should be a 2d (nested) list of matricess
```

Comment:
I'd suggest even at this stage we do a basic check for this. we know it needs to be group by reference time long and we have that information here. we also know it needs to be a D by D matrix